app/styles/front.scss

```diff
@@ -227,13 +227,16 @@ $autre-lighter: #cce4e7;
 
 
 /* Should be removed once all buttons have been converted to native html input fields. */
-.btn-default {
+.btn-default, .btn {
   &.active {
     background-color: $btn-success-bg;
-    color: white;
+    color: $btn-success-color;
+    .help-col {
+      color: #D8D8D8;
+    }
     &:focus, &:hover {
       background-color: $btn-success-bg;
-      color: white;
+      color: $btn-success-color;
```

Comment:
If we use a variable for the `background-color`, let's une one for the text color as well.